realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -230,6 +232,25 @@ private static synchronized void removeSession(SyncConfiguration syncConfigurati
         }
     }
 
+    /**
+     * Retruns the all valid sessions belonging to the user.
```

Comment:
Since `SynUser::all_sessions` checks whether each session is in error state, all sessions which are not in error state should be returned. So I think some sessions are not `active`.